e2e-tests/cypress/support/step_definitions/email_steps.ts

```diff
@@ -51,6 +51,21 @@ Then('the user receives an e-mail containing the {string} link', (linkName: stri
   )
 })
 
+Then('the user receives no password reset e-mail', () {
+  cy.origin(
+    Cypress.env('mailserverURL'),
+    { args: { userEMailSite } },
+    ({ userEMailSite }) => {      
+      cy.visit('/')
+      cy.get(userEMailSite.emailInbox).should('be.visible')
+
+      cy.get(userEMailSite.emailList)
+        .find('.email-item')
+        .should('have.length', 0)
```

Comment:
This seem not test the right length! Should be 1 cause we get an email for stephen@hawking.uk password reset!

Reply:
Fixed with 8553bf59fda699587bbb583be8a189dc35d43e77